app/user/runTest/finish.page.component.ts

```diff
@@ -0,0 +1,44 @@
+import {REACTIVE_FORM_DIRECTIVES} from "@angular/forms";
+import {Component, OnDestroy, OnInit} from "@angular/core";
+import {ROUTER_DIRECTIVES, Router, ActivatedRoute} from "@angular/router";
+import {Http} from "@angular/http";
+import {Constants} from "../../common/constants/constants.data";
```

Comment:
You will need to update stylejs.config to avoid such horrible pathes like "../../blabla" in the future.